### PR TITLE
suppress IllegalAccessError

### PR DESCRIPTION
**Pull request by jacekszymanski**
On Java 17 Builder fails to load nu.xom.JDK15XML1_0Parser with IllegalAccessError, as it is not suppressed, it never tries any further parsers.

This trivial PR suppresses the IllegalAccessError.

### CODE REVIEW

File: src/nu/xom/Builder.java [hunk 1]
```diff
@@ -188,64 +188,67 @@ public Builder(boolean validate, NodeFactory factory) {
...
         }
         catch (IllegalAccessException ex) {
             // look for next one
         }
         catch (NoClassDefFoundError err) {
-            // Xerces is not available; look for next one
-        } 
-        
+          // Xerces is not available; look for next one
+      }
```
Comment (elharo) [inline src/nu/xom/Builder.java:230]: The irony, it burns! Despite all the whitespace changes the indentation isn't the same here.

**Review by elharo — COMMENTED** [review]
Makes sense, I suppose. I may implement this myself just to avoid all the whitespace adjustments. Though longer term I should think about how to load that code anyway. Perhaps there's some way I can load it from a com package.

**Review by elharo — COMMENTED** [review]
Thanks for noticing this. I'll see what i can do.

File: src/nu/xom/Builder.java (hunk 1 above)
Comment (jacekszymanski) [reply src/nu/xom/Builder.java:230]: Seems like my VSCode setup is too smart for me. 🙈  I should have used vim...

Sorry for the whitespaces... the change is trivial anyway, so I think I won't bother you with another PR...

**Comment by elharo** [comment]
FYI, this code can simply be removed instead of catching an exception. Fix in progress.

**Pull request closed without merging**